### PR TITLE
add 4pin crystal

### PR DESCRIPTION
**Pull request by techmannih**
4pinCrystal
/claim #389

### CODE REVIEW

File: lib/components/normal-components/Crystal_4Pin.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+  }
+
+  initPorts() {
+    super.initPorts({
+      additionalAliases: {
+        pin1: ["pos", "top_left"],
+        pin2: ["neg", "top_right"],
+        pin3: ["cap1", "bottom_left"],
+        pin4: ["cap2", "bottom_right"],
```
Comment (imrishabh18) [inline lib/components/normal-components/Crystal_4Pin.ts:32]: The alias for this probably should be `top`, `bottom`, `left` and `right`, according to the symbol image

File: lib/components/normal-components/Crystal_4Pin.ts
Comment (imrishabh18) [inline lib/components/normal-components/Crystal_4Pin.ts:1]: I think probably we should use the `<Crystal />` component already present, and not introduce a new component just for when we are having 4pins?

cc: @seveibar

Comment (seveibar) [reply lib/components/normal-components/Crystal_4Pin.ts:1]: yes

**Review by seveibar — CHANGES_REQUESTED** [review]
see rishabh's comment

**Comment by imrishabh18** [comment]
@techmannih Please address the comments before asking for the review's

File: lib/components/normal-components/Crystal_4Pin.ts
Comment (seveibar) [reply lib/components/normal-components/Crystal_4Pin.ts:1]: No modify the existing Crystal to support the pinVariant prop

**Pull request closed without merging**